<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDL</class>
 <widget class="QMainWindow" name="QtDL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="URL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelURL">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter YouTube URL here:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuality">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality</string>
    </property>
   </widget>
   <widget class="QComboBox" name="qualityBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>64k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>96k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>160k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>192k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>228k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>288k</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>320k</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelFormat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Format</string>
    </property>
   </widget>
   <widget class="QComboBox" name="formatBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>85</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>AAC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FLAC</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>MP3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M4A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Opus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vorbis</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WAV</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Output folder</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pathButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose...</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>QtDL</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
